pkg/asset/installconfig/gcp/client.go

```diff
@@ -222,6 +224,34 @@ func (c *Client) GetProjects(ctx context.Context) (map[string]string, error) {
 	return projects, nil
 }
 
+// GetProjectRegions gets the regions that are valid for the project. An error is returned when unsuccessful
+func (c *Client) GetProjectRegions(ctx context.Context, project string) (map[string]string, error) {
+	ctx, cancel := context.WithTimeout(ctx, 1*time.Minute)
+	defer cancel()
+
+	svc, err := c.getComputeService(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	gctx := context.Background()
```

Comment:
nit: Probably best to just reuse the variable. Fine either way.
```suggestion
	ctx = context.Background()
```

Reply:
@rna-afk  I was told specifically not to reuse the context earlier. I can change this around though.

Reply:
My suggestion is to just reuse the variable and reset the context? I think we don't really need to use another variable but yeah it's just a nitpick.

Reply:
@rna-afk should we reset the variable since it was a parameter that was passed in ?